### DISCUSSION TITLE
alpha beta ratio of BED

### DISCUSSION THREAD

**Discussion by NatukiAdachi** [e0404/matRad#733] [Q&A]
I want to use the BED optimization , can I change the alpha beta ratio? If so, I would like to know how to change it.

**Comment by wahln** [comment]
The BED optimization on #534 has not yet been integrated / fully reviewed for integration into matRad, so you should use it with caution.

matRad stores the photon alpha/beta-ratios in the `cst`: 
`cst{i,5}.alphaX` is, for example, the photon alpha within the i-th structure.

The alpha/betas can be changed by changing this value, which should propagate to the BED optimization (@JenHardt implemented it, so she can confirm this). If you work with particles and variable RBE, however, situation is a bit more nuanced. For protons, this should work with any of the RBE models implemented on the dev branch, but for carbon ions, we rely on precomputed alpha/beta-dose kernels within the pencil-beam algorithm. The standard Generic Machine for Carbons only has two LEMIV-Kernels precomputed - one for alpha/beta = 0.1/0.05 = 2 and one for alpha/beta = 0.5/0.05 = 10, and these will the only alpha/beta values that will allow biological dose calculation and optimization with the generic carbon machine at the moment.

In the GUI, the alpha/beta ratio can be changed with the button "set tissue".

**Reply by NatukiAdachi** [reply]
I am able to successfully resolve the issue.
Thank you for your cooperation!!
